### PR TITLE
Allows runtime override of query and mutation consistency levels.

### LINKED ISSUE #729: Support more consistency levels in mutation options
Would it be possible to add more consistency levels to mutation options in [DmlSchemaBuilder](https://github.com/stargate/stargate/blob/f0c674b7d0d21301f027904ec33d0aeadd53f3a3/graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/DmlSchemaBuilder.java#L563-L568)?

The levels that are currently available are: LOCAL_ONE, LOCAL_QUORUM and ALL.
It would be nice to be able to use QUORUM, EACH_QUORUM, ONE, and ANY as well.

### PR DESCRIPTION
**Pull request by tub**
**What this PR does**:
Enables users to supply a list of consistency levels via the following system properties:
* `stargate.graphql.mutation_consistency_levels`
* `stargate.graphql.query_consistency_levels`

These values are comma separated, and if any given value is invalid it is skipped.
LOCAL_QUORUM is automatically added if it is omitted as it's the default consistency level.

If this approach looks sound I can add some documentation around the new parameters, and maybe add them to starctl.

**Which issue(s) this PR fixes**:
Fixes #729

**Checklist**
- [x] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated

### CODE REVIEW

File: graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/DmlSchemaBuilder.java [hunk 1]
```diff
@@ -64,16 +66,22 @@
...
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
+import org.apache.cassandra.stargate.db.ConsistencyLevel;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class DmlSchemaBuilder {
 
   private static final Logger LOG = LoggerFactory.getLogger(DmlSchemaBuilder.class);
+  public static final String STARGATE_MUTATION_CONSISTENCY_LEVELS =
```
Comment (tub) [inline graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/DmlSchemaBuilder.java:81]: Given that these are just for GraphQL we should probably rename to `stargate.graphql.mutation_consistency_levels` or similar

**Comment by jeffreyscarpenter** [comment]
@tub have you by chance completed the [DataStax CLA](https://cla.datastax.com/) process? (You may have already if you've contributed to other OSS projects sponsored by DataStax.) Our apologies for not having this step listed in our PR checklist, we'll work on fixing that.

**Comment by tub** [comment]
> @tub have you by chance completed the [DataStax CLA](https://cla.datastax.com/) process? (You may have already if you've contributed to other OSS projects sponsored by DataStax.) Our apologies for not having this step listed in our PR checklist, we'll work on fixing that.

Hey @jeffreyscarpenter, just running it past our legal folks & will get back to you.

**Comment by ivansenic** [comment]
@tub any update on this?

**Comment by tub** [comment]
> @tub any update on this?

Sorry for the delay, got it signed off, CLA is complete :)

**Comment by ivansenic** [comment]
@olim7t Can you provide a review here?

**Comment by ivansenic** [comment]
@tub this needs to be rebased on the latest master, some tests are failing because branch is not up-to-date.. Old issues with some tests..

File: graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/DmlSchemaBuilder.java [hunk 2]
```diff
@@ -64,16 +66,22 @@
...
+      "stargate.graphql.mutation_consistency_levels";
+  public static final String STARGATE_QUERY_CONSISTENCY_LEVELS =
+      "stargate.graphql.query_consistency_levels";
```
Comment (olim7t) [inline graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/DmlSchemaBuilder.java:84]: Nit: these could be private.

File: graphqlapi/src/test/java/io/stargate/graphql/schema/cqlfirst/dml/DmlSchemaOptionsTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+public class DmlSchemaOptionsTest {
+  @ParameterizedTest
+  @MethodSource
+  public void testConsistencyLevelParsing(String csvLevels, List<String> expectedValues) {
+    List<GraphQLEnumValueDefinition> values =
+        DmlSchemaBuilder.getConsistencyEnum(csvLevels, "Name").getValues();
+    // Are all values expected?
+    for (GraphQLEnumValueDefinition v : values) {
+      Assertions.assertTrue(expectedValues.contains(v.getName()));
```
Comment (olim7t) [inline graphqlapi/src/test/java/io/stargate/graphql/schema/cqlfirst/dml/DmlSchemaOptionsTest.java:20]: nit: for the sake of uniformity, could you use AssertJ assertions (`import static org.assertj.core.api.Assertions.assertThat`)?

That way lines 20 and 23 can be simplified as:
```
assertThat(expectedValues).contains(v.getName());
assertThat(values).hasSameSizeAs(expectedValues);
```
Or even better, replace lines 18-23 by:
```
assertThat(values)
    .extracting(GraphQLEnumValueDefinition::getName)
    .hasSameElementsAs(expectedValues);
```

**Review by olim7t — APPROVED** [review]
LGTM, just a couple of trivial cosmetic remarks.

**Pull request merged**